roles/create_tables/tasks/create_spec_managedclusterset_tracking_table.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: create managed_cluster_sets_tracking table
+  tags: tables
+  postgresql_table:
+    table: "{{ schema }}.{{ name }}"
+    db: "{{ hoh_db }}"
+    login_host: "{{ db_login_host }}"
+    login_user: "{{ db_login_user }}"
+    login_password: "{{ db_login_password }}"
+    ssl_mode: "{{ db_ssl_mode }}"
+    columns:
+      - cluster_set_name      varchar({{ cluster_name_length_limit }}) not null
+      - leaf_hub_name         varchar({{ cluster_name_length_limit }}) not null
+      - managed_clusters      jsonb not null default '[]'::jsonb # format : []string
+      - updated_at            timestamp not null default now()
+
+- name: create index on updated_at
+  postgresql_idx:
+    name: "{{ name }}_cluster_set_name_idx"
+    table: "{{ name }}"
+    schema: "{{ schema }}"
+    db: "{{ hoh_db }}"
+    login_host: "{{ db_login_host }}"
+    login_user: "{{ db_login_user }}"
+    login_password: "{{ db_login_password }}"
+    ssl_mode: "{{ db_ssl_mode }}"
+    columns: "updated_at"
+    idxtype: btree
+    unique: false
+
+- name: create index on cluster_set_name, leaf_hub_name
```

Comment:
questions:
1. do you have a use case where you select all MCS that are relevant for a LH?
2. do you have a use case where you select all rows matching a specific MCS?

I haven't looked in transport bridge, but I would guess the answer to question (1) is true - in order to know how to build a bundle per LH.

if the answer to question (2) is false, I think it would make sense to switch the order of the index - `leaf_hub_name` first, and then `cluster_set_name` - then we get for free index on LHs (each prefix on an index is also an index).